### PR TITLE
Port EnTT 3.10.2 to build2

### PR DESCRIPTION
**Pull request by Wmbat**
[CI](https://ci.cppget.org/@9b5992a8-0047-443d-aa80-5ffedda22828)

### CODE REVIEW

**Comment by Wmbat** [comment]
I'll do the bugfix release 3.10.3 right after this is merged and published !

**Comment by Klaim** [comment]
This package suffer from the issue of not using the symlink technique used now in all other packages. It could be fixed as part of this PR or in another PR depending on what you prefer, I will not be able to work on it immediately anyway.

**Comment by Wmbat** [comment]
I've made the changes to use symlinks like on all other package & the CI is running (the link has been updated)

I also noticed that the `EnTT` unit tests are not actually packaged yet. I'll open an issue for so that we don't forget about it. It uses `gtest` which we now have packaged in cppget.

File: entt/include [hunk 1]
```diff
@@ -0,0 +1 @@
+../upstream/src
```
Comment (Klaim) [inline entt/include:1]: You also need to add the `.gittattributes` with the line stating that symlink is a directory for windows support, see last paragraph of section 2 there: https://build2.org/article/symlinks.xhtml

**Comment by Klaim** [comment]
> I also noticed that the `EnTT` unit tests are not actually packaged yet. I'll open an issue for so that we don't forget about it. It uses `gtest` which we now have packaged in cppget.

Right, I thought I already made an issue for that package but apparently not. I did intended to add tests for that version but that can wait another PR.

**Comment by Klaim** [comment]
Sorry for the delay in merging this

**Pull request merged**